sklearn/model_selection/_search.py

```diff
@@ -1484,3 +1484,203 @@ def _run_search(self, evaluate_candidates):
         evaluate_candidates(ParameterSampler(
             self.param_distributions, self.n_iter,
             random_state=self.random_state))
+
+
+
+def OneSE(cv_results, **kwargs):
```

Comment:
We avoid **kw. Please define parameters explicitly. This is explained in our developers' guide